os_stub/openssllib/openssl_gen/der_digests_gen.c

```diff
@@ -0,0 +1,166 @@
+/**
```

Comment:
Since the `*conf.h` files have the DMTF license where appropriate, you don't need to add it to the generated files.

Reply:
done.

I moved openssl/configuration.h back to include dir, with DMTF license.
The file under openssl_gen does not have DMTF license now.